### ISSUE TITLE
IP Address (192.168.43.47)

### ISSUE THREAD

**Issue by mashunterbd** [kleo/android-portals#3]
My device's ip address is different.  Should I change the IP address in the script to my Android phone's IP or leave it as it is?
![Screenshot_2023-09-12-17-47-15-304_com termux](https://github.com/kleo/android-portals/assets/108648096/856e4408-d571-4b05-8616-31863b25da95)

**Comment by kleo** [comment]
Yes you should change it according to your phone's Mobile Hotspot IP address.
